rules-reviewed/eap7/eap6/eap6.windup.xml

```diff
@@ -140,5 +140,48 @@
                 </hint>
             </perform>
         </rule>
+        <rule id="eap6-08000">
+            <when>
+                <filecontent pattern="remote://{node}:{port}" filename="{*}.{extension}"/>
+            </when>
+            <perform>
+                <hint title="Remote JNDI Provider URL has changed in EAP 7" effort="1" severity="mandatory">
+                    <message>
+                        Default Remote JNDI Provider URL has changed in EAP 7. External applications using JNDI to lookup remote resources, for instance an EJB or a JMS Queue,
+                        may need to change the value for the JNDI InitialContext environment's property named `java.naming.provider.url`.
+                        The default URL scheme is now **http-remoting** instead of **remote**, and the default URL port is now **8080** instead of **4447**.
+                        
+                        As an example, consider the application server host is localhost, then clients previously accessing JBoss AS7 would use
```

Comment:
If you refer to EAP 7 elsewhere, you probably want to refer to EAP 6 here, instead of JBoss AS 7.